main.go

```diff
@@ -178,6 +178,10 @@ func runSuite(suite TestSuite) []TestResult {
 		vars := NewVars(hostFlag)
 		callArgsErr := vars.AddAll(testCase.Args)
 		for i, c := range testCase.Calls {
+			if c.On.Headers == nil {
+				c.On.Headers = map[string]string{}
+			}
+			c.On.Headers["X-Test-Name"] = testCase.Name
```

Comment:
you should change the header name to "drill-test-name".
@IgorKey , should add "drill-session-id" here?

Reply:
ya, both of them.
for first impl we need just put it with some constant value, e.g. "bozr-session"